### PR TITLE
EIP-draft: Submit Block RPC

### PR DESCRIPTION
**Pull request by BelfordZ**
This is a mirror of https://github.com/etclabscore/ECIPs/blob/master/ECIPs/ecip-1062.md

### CODE REVIEW

File: EIPS/eip-6969.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+eip: 6969
+title: Submit Block RPC
+author: Zachary Belford (@belfordz)
+status: Draft
+type: Standards Track
+category: Interface
+created: 2019-06-26
+---
```
Comment (axic) [inline EIPS/eip-6969.md]: This should depend on 1474.

File: EIPS/eip-6969.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Abstract
+At the moment, there is no high level facility through which an external-to-geth system can submit a newly mined block. Mining pools rely on submitWork, which implies that the provider of the submitted work is not able to, at the time of submission, provide an address to receive the payout. 
+
+By adding a JSON-RPC method that accepts a fully formed new block to submit instead of having to use `submitWork`, we will drastically simplify the process and software required to build a highly scalable mining pool. 
+
+## Motivation
+I would like to be able to submitWork, but have the address used be one other than the etherbase of the node the work is being submitted by.
+
+## Specification
+Below is the method specified using [OpenRPC](https://spec.open-rpc.org) spec.
```
Comment (axic) [inline EIPS/eip-6969.md]: Can you add a traditional description (too) which is used by the other EIPs for consistency?

File: EIPS/eip-6969.md (hunk 1 above)
Comment (BelfordZ) [reply EIPS/eip-6969.md]: Semi-related... have you seen OpenRPC or https://github.com/etclabscore/ethereum-json-rpc-specification ?

Comment (BelfordZ) [reply EIPS/eip-6969.md]: I've taken the liberty to update 1474 to something more maintainable.

https://github.com/BelfordZ/EIPs/blob/feat/json-rpc-spec/EIPS/eip-3333.md

File: EIPS/eip-6969.md (hunk 2 above)
Comment (BelfordZ) [reply EIPS/eip-6969.md]: I've added more of a description. I'm very open to suggestion / collaboration on this EIP. Let me know if this works. Im happy to try again :).

**Comment by BelfordZ** [comment]
Thx for the review. I believe I've addressed your comments.

**Comment by BelfordZ** [comment]
Re: https://github.com/etclabscore/go-ethereum/issues/24
https://github.com/etclabscore/ECIPs/blob/master/ECIPs/ecip-1062.md

**Comment by BelfordZ** [comment]
bump @axic

**Pull request closed without merging**